lib/actionview/all/actionview.rbi

```diff
@@ -1,5 +1,23 @@
 # typed: strong
 
+module ActionView
+  class ActionViewError < StandardError; end
+  class EncodingError < StandardError; end
+  class WrongEncodingError < EncodingError; end
+  class MissingTemplate < ActionViewError
+    sig { returns(String) }
+    def path; end
+  end
+
+  class Template
+    class Error < ActionViewError; end
+  end
+
+  TemplateError = T.type_alias(Template::Error)
```

Comment:
This can be just `TemplateError = Template::Error`, they are ~equivalent for class aliases